src/libasr/codegen/asr_to_fortran.cpp

```diff
@@ -462,7 +473,13 @@ class ASRToFortranVisitor : public ASR::BaseVisitor<ASRToFortranVisitor>
         s += "\n";
     }
 
-    // void visit_Exit(const ASR::Exit_t &x) {}
+    void visit_Exit(const ASR::Exit_t &x) {
+        s = indent + " exit";
```

Comment:
```suggestion
        s = indent + "exit";
```
(space is extra in the generated code)